### PR TITLE
Feat: add pyroscope

### PR DESCRIPTION
**Pull request by pj-cegeka**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,6 +22,30 @@ services:
...
+  jaeger:
+    image: jaegertracing/all-in-one:1.37
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
+    ports:
+      - "6831:6831/udp"
+      - "16686:16686"
+      - "4319:4317"
+      - "4320:4318"
+      - "14250:14250"
```
Comment (Yalz) [inline docker-compose.yml:49]: Would leave this out.. Grafana is definitely not essential for LDES Server config. and why Jaeger for your story?

File: ldes-server-instrumentation/pom.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <artifactId>ldes-server</artifactId>
+        <version>2.5.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ldes-server-instrumentation</artifactId>
+
+    <properties>
+        <maven.compiler.source>19</maven.compiler.source>
+        <maven.compiler.target>19</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (Yalz) [inline ldes-server-instrumentation/pom.xml:17]: best leave this out and let it inherit from parent

File: ldes-server-compaction/pom.xml [hunk 3]
```diff
@@ -101,6 +101,10 @@
             <groupId>org.junit.platform</groupId>
             <artifactId>junit-platform-suite</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.pyroscope</groupId>
+            <artifactId>agent</artifactId>
+        </dependency>
```
Comment (Yalz) [inline ldes-server-compaction/pom.xml:107]: needed for compaction too ??

**Review by Yalz — APPROVED** [review]

File: ldes-server-compaction/pom.xml (hunk 3 above)
Comment (pj-cegeka) [reply ldes-server-compaction/pom.xml:107]: I'll remove these in ingest as well. they where for the custom tags but running the server locally gives problems with them so I removed them

**Review by Yalz — APPROVED** [review]

**Pull request merged**